### ISSUE TITLE
Class distribution in Fever_NLI dataset

### ISSUE THREAD

**Issue by TRUMANCFY** [easonnie/combine-FEVER-NSMN#11]
Hi,

I had a glance on the fever_nli dataset, and noted that in the training set, there are stances from all three classes. However, there are only entailment/contradiction in the dev/test set.

Why is the dataset designed this way?

**Comment by easonnie** [comment]
Hi @TRUMANCFY,

You will need to retrieve the label for dev and test dataset by mapping `cid` to the original FEVER datasets. Please refer to the readme [here](https://github.com/easonnie/combine-FEVER-NSMN/blob/master/other_resources/nli_fever.md#what-is-the-format).
